README.md

```diff
@@ -0,0 +1,12 @@
+# ournetworks.ca
```

Comment:
Nice, a couple issues:
1. Could we add a one-sentence point to the planning repo, or link to the page or something for 2018?
1. We have to add a license & copyright line, needs to be assigned to individuals b/c we aren't an entity
1. I think best practice is we need to acknowledge the non-GPL and already licensed material (this includes normalize) and we should attribute to tomesh the 2017 material

Reply:
Also, RE: licensing, we need to add a blurb about site copy, I'd go the CC BY SA as we did that for the previous year (2017)

Reply:
@dcwalk Made edits from the feedback. Re: issue 2, I'm a little unsure on how to best approach it. Maybe this is something you can append to this PR?